### DISCUSSION TITLE
Problems running tests locally ?

### DISCUSSION THREAD

**Discussion by pablooliveira** [verificarlo/verificarlo#294] [Q&A]
@yohanchatelain When working on the recent PR I have had some failing tests locally that could be related to the recent changes in logger behavior. For example `test_ieee` fails because the expected log messages are not found by the check macro. 

But I am not sure if I should open an issue, since tests run fine on the CI.
Could you check if you can run the tests properly locally ? Thanks !

**Comment by yohanchatelain** [comment]
Hi @pablooliveira, I don't have issues on my side. Do you have the error message you got?

**Reply by pablooliveira** [reply]
Ok thanks, then it's probably something fishy on my side. I'll try to clean up and start over on my side, if I can reproduce it I will come back with the full log.
